src/index.js

```diff
@@ -266,7 +266,7 @@ class Backend {
             }
         } else {
             for (let i = 0; i < toWork.length; i++) {
-                fs.readFile(toWork[i].key, (error, data) => {
+                this.fs.readFile(toWork[i].key, "utf8", (error, data) => {
```

Comment:
Why did you add `this` here?

Reply:
I had an error during some tests regarding an unexisting fs at this line.
My bad I'll remove it.

Reply:
Done in commit https://github.com/reZach/i18next-electron-fs-backend/pull/4/commits/4728f30698228602ad05164e84b4832c99b95e73